### PR TITLE
chore: Fix README logo so it can be rendered in PyPI

### PR DESCRIPTION
**Pull request by edgarrmondragon**
The logo in the PyPI page doesn't look right.

It seems that the dialect of markdown GitHub supports is not 100% compatible with PyPI.

Ideally, there'd be a Poetry plugin similar to https://github.com/hynek/hatch-fancy-pypi-readme that would let us pre-process the README for PyPI. 

See also: https://github.com/pypi/warehouse/issues/11251

----
:books: Documentation preview :books:: https://meltano-sdk--1568.org.readthedocs.build/en/1568/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,13 +11,7 @@ de-facto open source standard for extract and load pipelines.
...
 </br>
 
-<div align="center">
-  <picture align="center">
-    <source media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/16805946/227660678-6e5fae45-a916-44f2-a222-379c76121584.svg">
-    <source media="(prefers-color-scheme: light)" srcset="https://user-images.githubusercontent.com/16805946/227660675-922ffca3-6b89-4d27-a3f2-a2a8ab2ebaf1.svg">
-  <img alt="Meltano Singer SDK Logo" src="https://user-images.githubusercontent.com/16805946/227660678-6e5fae45-a916-44f2-a222-379c76121584.svg" width="600"/>
-  </picture>
-</div>
+<img alt="Meltano Singer SDK Logo" src="https://user-images.githubusercontent.com/16805946/227660675-922ffca3-6b89-4d27-a3f2-a2a8ab2ebaf1.svg" width="600"/>
```
Comment (WillDaSilva) [inline README.md:14]: To support both light and dark mode with a single image, I think we should add a non-transparent background that contrasts with the colour of Melty and "meltano".

Comment (edgarrmondragon) [reply README.md:14]: Yeah, that'd be ideal

Comment (WillDaSilva) [inline README.md:14]: I've added a white background to the SVG:

227660675-922ffca3-6b89-4d27-a3f2-a2a8ab2ebaf1

```suggestion
<img alt="Meltano Singer SDK Logo" src="https://user-images.githubusercontent.com/11428666/231584532-ffa694e6-60f9-4fd6-b2ee-5ff3e39d3ad6.svg" width="600"/>
```

File: README.md [hunk 2]
```diff
@@ -11,13 +11,7 @@ de-facto open source standard for extract and load pipelines.
...
 </br>
 
-<div align="center">
-  <picture align="center">
-    <source media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/16805946/227660678-6e5fae45-a916-44f2-a222-379c76121584.svg">
-    <source media="(prefers-color-scheme: light)" srcset="https://user-images.githubusercontent.com/16805946/227660675-922ffca3-6b89-4d27-a3f2-a2a8ab2ebaf1.svg">
-  <img alt="Meltano Singer SDK Logo" src="https://user-images.githubusercontent.com/16805946/227660678-6e5fae45-a916-44f2-a222-379c76121584.svg" width="600"/>
-  </picture>
-</div>
+<img alt="Meltano Singer SDK Logo" src="https://user-images.githubusercontent.com/11428666/231584532-ffa694e6-60f9-4fd6-b2ee-5ff3e39d3ad6.svg" width="600"/>
```
Comment (edgarrmondragon) [inline README.md:14]:
```suggestion
<div align="center">
  <img alt="Meltano Singer SDK Logo" src="https://user-images.githubusercontent.com/11428666/231584532-ffa694e6-60f9-4fd6-b2ee-5ff3e39d3ad6.svg" width="600"/>
</div>
```

**Review by WillDaSilva — APPROVED** [review]

**Pull request merged**